src/table.js

```diff
@@ -291,11 +291,15 @@ export const addColumnAt = columnIndex => tr => {
 //   addRowAt(i, true)(state.tr)
 // );
 // ```
-export const addRowAt = (rowIndex, clonePreviousRow) => tr => {
+export const addRowAt = (rowIndex, cloneIndex) => tr => {
```

Comment:
now you need to fix some existing tests and add more tests for the new behaviour 😏